Source/Documentation/Manual/cabs.rst

```diff
@@ -16,8 +16,50 @@ OR supports with a high degree of compatibility all functions available in
 MSTS for 2D cabs, and provides some significant enhancements described in the 
 next paragraphs.
 
-OR adds support for the ETCS circular speed gauge, as described 
-:ref:`here <options-etcs>`.
+
+ETCS circular speed gauge
+-------------------------
+
+When this option is selected, it is possible to add to the cabview a
+circular speed gauge accordingly to the European standard train control
```

Comment:
The option got deleted from the code.

Reply:
Hi Peter,

In the Manual  had to move the text and the image from Options to Cabs. I have now removed that mention of "this option".

Thanks,

Chris